var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -354,6 +357,7 @@ class Gdal(CMakePackage, AutotoolsPackage, PythonExtension):
     # depends_on('ogdi', when='+ogdi')
     # depends_on('lib-opencad', when='+opencad')
     depends_on("opencl", when="+opencl")
+    # depends_on("libopendrive@0.6:", when="+opendrive")
```

Comment:
Should this PR be dependent on another PR that adds `libopendrive`, or will GDAL use a vendored copy if no version is found?

Reply:
No vendored copy. GDAL is an I/O library with ~200 drivers, each of which requires an external software dependency. Many but not all of those are already packaged in Spack. Technically, all of these commented out dependencies should also have a corresponding:
```python
conflicts("+opendrive")
```
to prevent the use of that variant until someone adds a package, I just haven't been bothering to do that. Let me know if you want me to add conflicts (or comment out variants).